event/src/models/User.ts

```diff
@@ -33,6 +37,19 @@ const UserSchema = new mongoose.Schema<IUsers>({
     type: Boolean,
     default: false,
   },
+  profile_pic: {
+    type: String,
+    required: true,
```

Comment:
Instead of requiring a profile picture, I think it's better to default to a placeholder instead, then user can upload later

```suggestion
    default: "/placeholder/image",
```